docs/user/openstack/README.md

```diff
@@ -33,7 +37,51 @@ clouds:
       auth_url: 'https://10.10.14.22:5001/v2.0'
 ```
 
-* Swift must be enabled.  The user must have `swiftoperator` permissions and
+If you choose to use environment variables in place of a clouds.yaml, or along side it, consult the following doccumentation:
+https://www.terraform.io/docs/providers/openstack/#configuration-reference
+
+
+## OpenStack Requirements
+
+### Recommended Minimums
+
+With the latest version of the installer, we recommend the following minimums in order to get a succesful deployment.
+
+   * OpenStack Quota
+     * Floating IPs
+       * Minimum: 2
+       * Recommended: 3
+     * Security Groups
+       * Minimum: 3
+       * Recommended: 8
+     * Security Group Rules
+       * Minimum: 60
+       * Recommended: 100
+     * Routers
+       * Minimum: 1
+     * Subnets
+       * Minimum: 1
+     * RAM
+       * Minimum: 80 Gb
+     * Instances
+       * Minimum: 5
```

Comment:
We've got 1 bootstrap, 3 masters and 2 workers minimum. That's six. Could you increase this and the recommended value by one please?

Reply:
LOL whoops

Reply:
Users could configure more compute, etc.  It seems like you should either emphasize that these minimums are for the installer defaults, or break them down by install-cinfig tunables.

Reply:
@wking I tired to clarify things in my latest commit. Please let me know if my changes have resolved this issue